app/src/main/res/anim/right_to_left.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<set xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shareInterpolator="false">
+    <translate android:fromXDelta="-60%" android:toXDelta="0%"
+        android:fromYDelta="0%" android:toYDelta="0%"
+        android:duration="150"/>
+</set>
```

Comment:
POSIX standards indicate that EOF should preferably be a new line. Please look into it.